hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/util/ConfigUtils.java

```diff
@@ -23,12 +23,11 @@
 import org.apache.hudi.common.util.StringUtils;
 
 public class ConfigUtils {
-
-  public static final String SPARK_QUERY_TYPE_KEY = "spark.query.type.key";
-
-  public static final String SPARK_QUERY_AS_RO_KEY = "spark.query.as.ro.key";
-
-  public static final String SPARK_QUERY_AS_RT_KEY = "spark.query.as.rt.key";
+  /**
+   * Config stored in hive serde properties to tell query engine (spark/flink) to
+   * read the table as a read-optimized table when this config is true.
+   */
+  public static final String IS_QUERY_AS_RO_TABLE = "hoodie.query.as.ro.table";
 
```

Comment:
What's the relationship between this key `IS_QUERY_AS_RO_TABLE` and `SPARK_QUERY_AS_RO_KEY ` and `SPARK_QUERY_AS_RT_KEY `.

Reply:
Then we should add some comments and some strategy to deprecate (or delete) them.

Reply:
This  PR #2925 has not released yet, so we can just delete them.